lib/widgets/twake_components/twake_loading/twake_loading_indicator.dart

```diff
@@ -2,23 +2,23 @@ import 'package:fluffychat/widgets/twake_components/twake_loading/status_loading
 import 'package:flutter/widgets.dart';
 import 'package:flutter/cupertino.dart';
 
-class TypingIndicator extends StatefulWidget {
-  const TypingIndicator({
+class TwakeLoadingIndicator extends StatefulWidget {

```

Comment:
Can we do it as a `StatelessWidget`?

Reply:
Because it's an animation widget with differents states, I followed the official tutorial https://docs.flutter.dev/cookbook/effects/typing-indicator